row_039_uos.js

```
@@ -0,0 +1,2484 @@
+
+                connection.end();
+
+            });
+            break;
+        }
+
+        // 회원가입 - UOS파트너
+        case '0002': {
+            var check_overlap_text = "select * from uospartner_account where id=?";
+            var res_data_string ='';
+            connection.query(check_overlap_text, message.uospartner_id , function(err, result, fields){
```

Biome (2.1.2)

[error] Shouldn't redeclare 'check_overlap_text'. Consider to delete it or rename it.

'check_overlap_text' is defined here:


(lint/suspicious/noRedeclare)